#!/usr/bin/env python
PACKAGE = "stargazer_ros_tool"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("debug_mode", bool_t, 0, "Activate debug mode for landmark localizer to show results", False)

exit(gen.generate(PACKAGE, "stargazer_ros_tool", "LandmarkLocalizer"))
